### PR TITLE
Removing misleading server time text, and associated route, controller method, and test.

### PR DESCRIPTION
**Pull request by YujinC**
I am removing the misleading "Server time" information under the student assignment "Submissions" tab and the corresponding javascript, route, and controller method.

## Motivation and Context

This pull request removes some unnecessary information (server time) because it is updated through polling only.

## Your Changes

**Description**:
I removed an ajax request and its corresponding partial which displayed the local server time, along with the matching route and controller method.

**Type of change** (select all that apply):

 

- [X] Bug fix (non-breaking change which fixes an issue)

## Testing

I manually tested this by logging in as a student, going to an assignment, then clicking the submissions tab.

## Questions and Comments (if applicable)

~Should I modify the yml in any way (for the localizations)?~ Turns out I should.

## Checklist

- [X] I have performed a self-review of my own code.
- [x] I have fixed any Hound bot comments. 
- [x] I have verified that the TravisCI tests have passed. 
- [ ]  I have updated the Changelog.md file.
- [x] I have reviewed the test coverage changes reported on Coveralls.

### CODE REVIEW

**Review by david-yz-liu — COMMENTED** [review]
Good work. Two comments:

1. You can remove the `server_time?` authorization method from `app/policies/submission_policy.rb`, and the corresponding strings from `locales/policies`.
2. This change warrants an entry in the Changelog.

**Comment by YujinC** [comment]
Modified the respective files. Is my changelog comment adequate? I copied the formats of the others and put my student id. Not sure if that's the correct format.

File: Changelog.md [hunk 1]
```diff
@@ -371,3 +371,4 @@
 - Fixed annotation_category bug, and average calculation bug (c6conley)
 - Closing #402 (c6conley)
 - Add version and patch level information to MarkUs (g9jerboa)
+- Removed server_time information in submissions_controller.rb and server_time? from submission_policy.rb (choyuji3)
```
Comment (david-yz-liu) [inline Changelog.md:374]: Oh this is the old style here. You should add the entry to the bottom of the "unreleased" section near the top of the file, and include the PR number.

Comment (YujinC) [reply Changelog.md:374]: Ohhh, alright. Thank you.

**Pull request merged**